core/domain/stats_jobs_one_off.py

```diff
@@ -613,12 +632,12 @@ def reduce(exp_id, values):
 
             exp_stats_dicts.append(copy.deepcopy(exp_stats_dict))
 
-        for idx, stats_dict in enumerate(exp_stats_dicts):
-            if stats_dict != old_stats_dicts[idx]:
+            old_stats_dict = old_stats[version - 1].to_dict()
+            if exp_stats_dict != old_stats_dict:
                 yield (
                     'ERROR in recomputation. exp_id: %s, exp_version: %s, '
                     'new_stats_dict: %s, old_stats_dict: %s' % (
-                        exp_id, idx + 1, stats_dict, old_stats_dicts[idx]))
+                        exp_id, version, exp_stats_dict, old_stats_dict))
```

Comment:
Yield this only for exploration_id '1' ? and instead of old stats, yield the list of exploration state names and the new stats only! Thanks